demisto_sdk/commands/test_content/xsiam_tools/xsiam_client.py

```diff
@@ -126,6 +126,16 @@ def _session(self) -> requests.Session:
     def _session(self, value: requests.Session):
         self.__session = value
 
+    def get_xsiam_version(self) -> str:
+        endpoint = urljoin(self.base_url, 'xsoar/about')
+        response = self._session.get(endpoint)
+        response.raise_for_status()
+        data = response.json()
+        xsiam_version = data.get('demistoVersion', '99.99.99')
```

Comment:
please use: `DEFAULT_CONTENT_ITEM_TO_VERSION` from demisto_sdk/commands/common/constants.py instead of '99.99.99'